packages/integration-tests/src/tests/api/admin-user.roles.test.ts

```diff
@@ -26,10 +34,10 @@ describe('admin console user management (roles)', () => {
     const role = await createRole({});
 
     await assignRolesToUser(user.id, [role.id]);
-    await expectRejects(assignRolesToUser(user.id, [role.id]), {
-      code: 'user.role_exists',
-      statusCode: 422,
-    });
+    await assignRolesToUser(user.id, [role.id]);
+    const roles = await getUserRoles(user.id);
```

Comment:
the original case of "role_exists" is deleted?

Reply:
Deleted, silently skip those existing roles.